src/tests/ftest/util/verify_perms.py

```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/env python3
```

Comment:
This file is executed as a command in tests, so this shouldn't be removed.

Reply:
Done